_posts/2020-10-01-hacktoberfest.markdown

```diff
@@ -0,0 +1,70 @@
+being dismissed, so please color within the lines.
+
+The first area is for non-code contributions.  We maintain all of our
+documentation using a tool called
+[mdBook](https://github.com/rust-lang/mdBook).  This has the net
+effect that all of our documentation is written as version controlled
+markdown files.  If you regularly use Void and have been looking for a
+place to get involved, this is a good point to jump in.  We aren't
+trying to recreate the Arch wiki though, so make sure you understand
+the scope and purpose of the docs repo before sending your
+documentation.
+
+For technical contributions, we are accepting updates of existing
+packages.  While new packages may count as well, we have had many
+problems in the past with people adding packages and then abandoning
+them, so expect to meet significant resistance if you are a new
+contributor trying to add a package to the repo.  Updating packages is
+very easy.  You can select a package from the list of [out of date
+packages](http://alpha.de.repo.voidlinux.org/void-updates/void-updates.txt)
+and update it using the tools in the [void-packages
+repo](https://github.com/void-linux/void-packages).  The
+[manual](https://github.com/void-linux/void-packages/blob/master/Manual.md)
+might be of assistance when you are updating packages.
+
+As a general rule, we would prefer Hacktoberfest contributors that do
+not have a previous track record with the Void Linux project steer
+clear of "structural" packages unless you have specific domain
+knowledge that qualifies you to work on high-risk packages.  When
+selecting a package to update, select a package registered to
+`orphan@voidlinux.org` when possible.  These packages are otherwise
+unmaintained, and your contribution will have a bigger impact.  You
+can update packages that are owned by existing developers, but
+understand that conflicts between a maintainer and contributor will be
+resolved at the discretion of Void staff.
+
+Here are some useful tips when updating packages:
+
+  * Don't PR broken code, our maintainers are much less likely to give
+    a second look to a PR that didn't build when it was submitted.
+  * While its possible to run void-packages from an alien distro, this
+    isn't really supported, if you're a seasonsed Linux user and want
+    to try Void, now is the time!
+  * The docs use versioned markdown, you can run this locally to
+    format your files.
+  * The update list is sometimes wrong, we'd love to get patches that
+    improve its reliability by ignoring beta versions or adding checks
+    to packages that are not detected as out of date correctly.
+  * If you have expertise in C, GNU Autotools, or other build systems,
+    taking a look at projects that we've marked as not
+    cross-compilation compatible and fixing the upstream issue can be
+    an amazing contribution that impacts more than just Void.
+
+We look forward to working with amazing world of open source
```

Comment:
`We look forward to working with amazing world of open source`

Missing `an` or `the` after with.